src/hooks/useIntervalSafe.tsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useRef } from 'react';
+import useAsyncEffect from 'use-async-effect';
+
+export const useIntervalSafe = (
+  callback: (isMounted: () => boolean) => void,
+  delay: null | number,
+  leading = true
+) => {
+  const savedCallback = useRef<(isMounted: () => boolean) => void>();
+
+  useEffect(() => {
+    savedCallback.current = callback;
+  }, [callback]);
+
+  useAsyncEffect<number | undefined>(
+    async (isMounted) => {
+        function tick() {
```

Comment:
sorry for nit picking 🙄 

lint and prettify on save. 

consistent use of arrow functions.

Reply:
nothing to sorry about - my bad, fixed (it was based on the useInterval...)